inc/deploygroup_staticdata.class.php

```diff
@@ -145,6 +145,14 @@ public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $
     */
     public static function showCriteriaAndSearch(PluginGlpiinventoryDeployGroup $item)
     {
+
+        echo "<div class='alert alert-primary d-flex align-items-center' role='alert'>";
+        echo "<i class='fas fa-info-circle fa-xl'></i>";
+        echo "<span class='ms-2'>";
+        echo __('Make a search to get desired computer, then use massive actions and use <strong>Add to static group</strong>');
```

Comment:
Avoiding html tags is maybe better, you should use sprintf, and reuse the existing "Add to static group" string. Also, you've missed the i18n domain